03/app.js

```diff
@@ -2,4 +2,17 @@ document.addEventListener('DOMContentLoaded', init);
 
 function init() {
     console.log('DOM');
+    const btn = document.querySelector('button');
+    const span = document.querySelector('span');
+
+    const getIP = () => {
+        const url = 'https://api.ipify.org?format=json';
+        fetch(url)
+            .then(response => {
```

Comment:
Warto też sprawdzić czy `response.ok === true`:
https://dev.to/myogeshchavan97/do-you-know-why-we-check-for-response-ok-while-using-fetch-1mkd